knowledge/science/botany/jasminum/qna.yaml

```diff
@@ -0,0 +1,19 @@
+created_by: 
+domain: flowers
+seed_examples:
+- answer:  red
+  question: redhat color
+- answer: |The rose's botanical name is Rosa rubiginosa. There are roughly 360 species of roses in the genus Rosa and Rosaceae.
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!